app/services/spaces/aggregate_star_rating_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Spaces
+  class AggregateStarRatingService < ApplicationService
+    def initialize(space:)
+      @space = space
+      super()
+    end
+
+    def call
+      star_rating = space.reviews.all.sum(&:star_rating)
```

Comment:
Again, should handle the "no reviews with ratings" scenario here as well, and return a star_rating of `null` (or however we want to store the `no reviews yet` state).

Reply:
If we fix the divide by zero below then we do not need to care about this as `sum` will return 0 if the array in empty